src/readers/morphologyASC.cpp

```diff
@@ -257,8 +255,8 @@ class NeurolucidaParser
     ErrorMessages err_;
 };
 
-Property::Properties load(const std::string& uri, unsigned int options) {
-    NeurolucidaParser parser(uri);
+Property::Properties load(const std::string&& input, const std::string& uri, unsigned int options) {
+    NeurolucidaParser parser(std::move(input), uri);
```

Comment:
I don't understand why I need to use `std::move` here since input is already an rvalue reference and NeurolucidaParser first argument also expect an rvalue reference...

Reply:
Assuming that `load`, if passed a `std:string` is `asc` seems like we'll be boxing ourself into a corner when we want to support swc, for instance.